registry/auth.go

```diff
@@ -180,6 +180,8 @@ func loginV1(authConfig *cliconfig.AuthConfig, registryEndpoint *Endpoint) (stri
 				}
 				// *TODO: Use registry configuration to determine what this says, if anything?
 				return "", fmt.Errorf("Login: Account is not Active. Please see the documentation of the registry %s for instructions how to activate it.", serverAddress)
+			} else if resp.StatusCode == 500 { // Issue #14326
+				return "", fmt.Errorf("Internal Server Error")
```

Comment:
Please provide more information in a log output. Bowling over the 500 error with no context will make these impossible to debug at a later date.